exec/java-exec/src/test/java/org/apache/drill/exec/store/dfs/TestFileSelection.java

```diff
@@ -63,4 +63,17 @@ public void testEmptyFolderThrowsTableNotFound() throws Exception {
     }
   }
 
```

Comment:
Maybe it makes sense to move new unit tests and `testEmptyFolderThrowsTableNotFound` into separate class which will test behavior when incorrect object is defined (i.e. schema, workspace, table)?

Reply:
done